sprout/appliances/models.py

```diff
@@ -1171,7 +1171,10 @@ def clone(self, num_appliances=None, time_leased=60, owner=None):
     def create(cls, owner, group, version=None, date=None, provider=None, num_appliances=1,
             time_leased=60, preconfigured=True, yum_update=False, container=False, ram=None,
             cpu=None, provider_type=None):
-        container_q = ~Q(container=None) if container else Q(container=None)
+        if container:
+            container_q = ~Q(container=None) & ~Q(provider_type='openshift')
```

Comment:
This will not work as expected, because provider_type is not an obligatory parameter.

Reply:
provider_type is mandatory now. thanks @mfalesni